collectd-etcd/README.md

```diff
@@ -0,0 +1,128 @@
+#![](https://github.com/signalfx/integrations/blob/master/collectd-etcd/img/integrations_etcd.png) etcd
```

Comment:
We need a space here between the `#` and the `!`.  I suspect this is copied from Elasticsearch which is also incorrect.  With out the space the text will not be set as a header.  See docker plugin's [readme](https://raw.githubusercontent.com/signalfx/integrations/master/collectd-docker/README.md) as an example